evm_arithmetization/Cargo.toml

```diff
@@ -56,6 +56,7 @@ sha2 = "0.10.6"
 [features]
 default = ["parallel"]
 asmtools = ["hex"]
+jemalloc_disable = []
```

Comment:
Tiniest nit: maybe swap words? Or oppositely, have a `jemalloc` feature that activates the allocator.

Reply:
Yeah I'm fine with that.

Reply:
Ended up sticking with the negative (using the feature stops the custom allocator) just because I'm going with the assumption that `jemalloc` is slightly better for now until we know.